src/xhr.js

```diff
@@ -59,6 +61,35 @@ class Xhr {
       return assertSuccess(response).json();
     });
   }
+
+  postJson(input, init) {
```

Comment:
Do we need `postJson`? Shouldn't the `fetchJson(..., {method: 'POST'})` already do this? Those two should be the same, but currently will go over two very different code paths.